dark-facebook-messenger-theme.user.css

```diff
@@ -23,14 +23,35 @@
         color: #1a1a1a;
     }
 
-    ._4sp8, .clearfix._17pz {
+    /*Normal texts, originally black text on white background*/
+    /*._29_7 ._3058._s1-._6e6n : "... replied to you" / "You replied to..." boxes */
+    ._4sp8, .clearfix._17pz, ._3egs, ._3058, ._29_7 ._3058._s1-._6e6n {
         color: var(--normal-dark-red);
         background-color: var(--dark-gray);
     }
+    
+    /*_8rsr: like button*/
+    ._8rsr {
+        fill: var(--dark-gray);
+    }
+    
+    /* Selected person */
+    ._1ht1._1ht2  ._6zkc {
+        background-color: rgba(0, 0, 0, .5);
+    }
```

Comment:
This is also a good change! 👍 But I think it makes too much contrast in respect to the rest of the theme. Therefore changing the value of the transparency from `.5` to something like `.15` would still make the selected chat more visible.